src/components/TextInputFocusable/index.js

```diff
@@ -267,43 +268,8 @@ class TextInputFocusable extends React.Component {
         // If paste contains HTML
         if (types.includes(TEXT_HTML)) {
             const pastedHTML = event.clipboardData.getData(TEXT_HTML);
-
             event.preventDefault();
-            const domparser = new DOMParser();
-            const embeddedImages = domparser.parseFromString(pastedHTML, TEXT_HTML).images;
-
-            // If HTML has img tag, then fetch images from it.
-            if (embeddedImages.length > 0) {
-                fetch(embeddedImages[0].src)
-                    .then((response) => {
-                        if (!response.ok) { throw Error(response.statusText); }
-                        return response.blob();
-                    })
-                    .then((x) => {
-                        const extension = IMAGE_EXTENSIONS[x.type];
-                        if (!extension) {
-                            throw new Error(this.props.translate('textInputFocusable.noExtentionFoundForMimeType'));
-                        }
-
-                        return new File([x], `pasted_image.${extension}`, {});
-                    })
-                    .then(this.props.onPasteFile)
-                    .catch(() => {
-                        const errorDesc = this.props.translate('textInputFocusable.problemGettingImageYouPasted');
-                        Growl.error(errorDesc);
-
-                        /*
-                        * Since we intercepted the user-triggered paste event to check for attachments,
-                        * we need to manually set the value and call the `onChangeText` handler.
-                        * Synthetically-triggered paste events do not affect the document's contents.
-                        * See https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event for more details.
-                        */
-                        this.handlePastedHTML(pastedHTML);
-                    });
-                return;
-            }
-
-            this.handlePastedHTML(pastedHTML);
+            this.handlePastedHTML(this.stripImageTags(pastedHTML));
```

Comment:
@mananjadhav I don't think you need to strip image tags here. They are automatically stripped

Reply:
Good point! @mananjadhav try to test it without the `stripImageTags`.

Reply:
Thanks. Still worked without `stripImageTags`